frontend/src/stores/levels/actions.ts

```diff
@@ -0,0 +1,47 @@
+import { get, post, put, deleteRequest } from '@/services/http'
+import { levels } from './state'
+import type { Level } from '@/domain/Level'
+
+// Function to sort levels based on minCoefficient and maxCoefficient
+function sortLevels(levels: Level[]): Level[] {
+  return levels.slice().sort((a, b) => {
+    return a.minCoefficient - b.minCoefficient || a.maxCoefficient - b.maxCoefficient
+  })
+}
+
+async function fetchLevels(): Promise<void> {
+  if (levels.value.length > 0) {
+    return
+  }
+
+  const fetchedLevels = await get<Level[]>('/level')
+  setLevels(fetchedLevels)
+}
+
+function setLevels(newLevels: Level[]): void {
+  // Sort fetched levels before setting them
+  levels.value = sortLevels(newLevels)
+}
+
+function removeLevel(levelId: number) {
```

Comment:
Your ticket is about **list the levels of a role**. Why have you created functions to remove, save and edit???